great-idea/index.html

```diff
@@ -13,11 +13,81 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
-
   <body>
-
     <!-- Copy and paste your HTML from the first UI project here -->
+    <div class="container">
+      <header>
+        <nav class="navigation">
```

Comment:
Be aware of unused class tags. It's a good practice to markup classes as you need them.